README.md

```diff
@@ -33,14 +33,20 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the difference between an adaptive website and a fully responsive website?
+Responsive website changes the size of the web page to adapt to the user's screen while adaptive design creates different web pages for different devices.
 
 2. Describe what it means to be mobile first vs desktop first.
+Mobile first is when you design a website to fit mobile screens first which is also the content first strategy before designing to fit other screen sizes.
```

Comment:
Mobile-first designs the layout for mobile first and then adjusts the layouts for larger screens **(using min-width)**. while the reverse is the case for desktop-first **(using max-width)**